operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/DefaultControllerConfiguration.java

```diff
@@ -54,34 +54,6 @@ public DefaultControllerConfiguration(
     setConfigurationService(service);
   }
 
-  /**
```

Comment:
Please keep this, we'll remove it in v2, it's there for backwards compatibility.

Reply:
Ahh, ok, will revert this and remove the javadoc

Reply:
done.